auth/basics/js.md

```diff
@@ -105,6 +107,8 @@ skygear.auth.signupWithUsername(username, password).then((user) => {
 
 ``` javascript
 import skygear from 'skygear';
+let email = "hello@skygear.io";
+let password = "1234";
```

Comment:
It is good to have comment `// very secure password`, but seems some of the password in this PR miss it. Can you help add them all?